docs/themes/oso-tailwind/layouts/partials/header.html

```diff
@@ -3,7 +3,7 @@
     <div id="header-page-title" class="flex-grow">
         <p class="text-white font-base text-sm  text-center truncate">
             {{ if eq $.Page.Language.Lang "python" }}
```

Comment:
should this be `"python"` or `"node"` now?

Reply:
I think we're reverting this to slack